tests/TestCase/Command/BakeMigrationDiffCommandTest.php

```diff
@@ -57,10 +88,23 @@ public function tearDown(): void
                 unlink($file);
             }
         }
+
+        // Clean up any TheDiff migration files from all directories
+        $configPath = ROOT . DS . 'config' . DS;
+        $directories = glob($configPath . '*', GLOB_ONLYDIR) ?: [];
+        foreach ($directories as $dir) {
+            $migrationFiles = glob($dir . DS . '*TheDiff*.php') ?: [];
```

Comment:
Do you need to clean up in both setup and teardown? Doing it in setup should be good enough as you're relying on glob() and not object properties.

Reply:
I was fiddling around for hours, with different setups and DBs and always somewhere there was some fail.
But I will try to remove the teardown part and see if it works locally and remote :)